modules/EasyAbp.EShop.Stores/src/EasyAbp.EShop.Stores.Application/EasyAbp/EShop/Stores/StoreOwners/StoreOwnerAppService.cs

```diff
@@ -0,0 +1,54 @@
+﻿using EasyAbp.EShop.Stores.Permissions;
+using EasyAbp.EShop.Stores.StoreOwners.Dtos;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Volo.Abp;
+using Volo.Abp.Application.Services;
+
+namespace EasyAbp.EShop.Stores.StoreOwners
+{
+    public class StoreOwnerAppService : ReadOnlyAppService<StoreOwner, StoreOwnerDto, Guid, GetStoreOwnerListDto>,
+        IStoreOwnerAppService
+    {
+        protected override string GetListPolicyName { get; set; } = StoresPermissions.Stores.Default;
```

Comment:
1. What about the "get" policy?
2. Should people who have `StoresPermissions.Stores.Default` be allowed to get other stores' and users' StoreOwner?